projects/ccf-rui/src/app/shared/components/details-label/details-label.component.theme.scss

```diff
@@ -0,0 +1,9 @@
+@use '~@angular/material/theming' as *;
+
+@mixin theme($theme) {
+  $foreground: map-get($theme, foreground);
+
+  .label-container {
```

Comment:
This should be wrapped inside a `.ccf-details-label` block to prevent it from affecting other components. Look at the theming and code for other components (right-sidebar, rotation-slider, etc.) for how it is structured.